### ISSUE TITLE
support web install

### ISSUE THREAD

**Issue by RelicOfTesla** [kubesphere/kubekey#1193]
### Describe this feature

支持通过网页来填写配置，安装，而不是手工上去改配置文件，然后一个个指令去打，太麻烦。
网页安装好处就是可以流程化，step by step 配置不同选项。
表单UI安装是非常友好的选项。。
现在最新版的go支持html资源内陷到二进制执行程序中(go:embed)，也是非常方便做web了。

### Describe the solution you'd like

例如
./kk  web_install [--port=xxx]
监听 http://:xxxx网页
[ 可选：打印出安装访问密码 xxxxxxx (随机)，访问安装程序时需要输入这个安装密码 ]

然后让用户访问网页，进行安装。。

**Comment by pixiake** [comment]
Yes, we are considering supporting cluster installation through ui.
